testing/tests/filter_block.rs

```diff
@@ -128,3 +128,22 @@ fn filter_block_not_html_escape() {
     let template = E;
     assert_eq!(template.render().unwrap(), r#"<block>"#);
 }
+
+// This test checks that the filter chaining is working as expected.
+#[derive(Template)]
+#[template(
+    source = r#"{% filter lower|indent(2)|capitalize -%}
+HELLO
+{{u}}
+{%- endfilter %}"#,
+    ext = "txt"
+)]
+struct F {
+    u: &'static str,
+}
+
+#[test]
+fn filter_block_chaining() {
+    let template = F { u: "yOU" };
```

Comment:
A friend by reading this told me it could be badly interpreted. Didn't pay attention, sorry about that. Changing right away.

Reply:
I work at the FU Berlin (Freie Universität, Free University). I'm desensitized :D

Reply:
Didn't expect this comeback :laughing: